pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
> I don't remember why it was added. I think @dakrone might know.

I added it as a way to not be dependent on a Java IDE to run a bootstrap version of ES (not everyone wants to use Intellij or Eclipse!), so as long as there is a way to run the checkout from the command line, I am happy.

> Somehow this isn't bothering anyone all the output is "missing" ..., but it makes things undebuggable.

It's logging everything to `logs/elasticsearch.log`, I agree it's less than ideal.

+1 on any plugin that does this that is easier/nicer than the maven-exec one. antrun sounds like it would work great?